src/swap/types.ts

```diff
@@ -103,8 +105,9 @@ export type LifecycleStatus =
   | {
       statusName: 'transactionApproved';
       statusData: {
+        callsId?: Hex;
         transactionHash?: Hex;
-        transactionType: 'ERC20' | 'Permit2' | 'Batched';
+        transactionType: 'Batched' | 'ERC20' | 'Permit2' | 'Swap';
```

Comment:
Where are we documenting the difference between them?

I feel like we should add some comments here to expand how those types are different.

Reply:
added comment documenting the different types